ejercicio1/form.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="author" content="bautistq">
+	<meta name="description" content="Sign up form">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Sign up form</title>
+</head>
+<body>
+	<section>
+		<h1>Form</h1>
+			<form>				
+				<label for="name">First Name</label>
+				<input type="text" name="fname" id="fname" required="requiered">
+
+				<label for="lname">Last Name</label>
+				<input type="text" name="lname"> 
+
+				<label for="email">Email</label>
+				<input type="email" name="email" required="required"> 
+
+				<label for="sport">Sport</label>
+				<select name="sport">
+				 	<option value="Soccer">Soccer</option>
+				 	<option value="Baseboll">Baseball</option>
+				 	<option value="Cycling">Cycling</option>
+				 	<option value="Swimming">Swimming</option>				 	
+				 </select>
+
+				 <label for="bio">About you</label>				 
+				 <textarea name="bio"></textarea>
+				 
+				<button name="send" id="send">Send</button>
+				<button name="borrar" id="borrar">Borrar</button>
```

Comment:
The `button` tag basically do the same as `input` with type submit. That's why both buttons submits the form. I'd use a button and a input with type reset